python/graphstorm/model/embed.py

```diff
@@ -179,6 +179,19 @@ def __init__(self,
         self.dropout = nn.Dropout(dropout)
         self.use_node_embeddings = use_node_embeddings
         self.activation = activation
+
+        # NCCL backend is not supported for utilizing learnable embeddings on nodes. It has a
+        # dependency on distDGL (PR https://github.com/dmlc/dgl/pull/5929).
+        # TODO (Israt): Add NCCL support to distDGL.
+        if th.distributed.is_initialized() and th.distributed.get_backend() == "nccl":
```

Comment:
Please use: https://github.com/awslabs/graphstorm/blob/b199d281e0de6e8bf25509507d788fb57a7a1f3f/python/graphstorm/utils.py#L43-L45

Reply:
We should also add a get_backend in graphstrom.utils.